addons/account_audit_trail/report/audit_trail_report_views.xml

```diff
@@ -27,7 +27,7 @@
                 <field name="account_audit_log_move_id"/>
                 <field name="author_id"/>
                 <field name="date" string="Date"/>
-                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]"/>
+                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
```

Comment:
Why this group and not `group_account_manager` (Accountant) or `group_account_user` (Bookkeeper) for instance ?

Reply:
So, we need `base.group_system` because the field is defined with this group, so we need it to be able to access it.
The groups defined in accounting don't have `base.group_system`, so we can't use them